docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
-Sphinx
+Sphinx!=1.8.0
 git+https://github.com/fabpot/sphinx-php.git
-sphinx_rtd_theme
\ No newline at end of file
+sphinx_rtd_theme
```

Comment:
the dev kit changes should be available as they get executed against `master` separately

Reply:
What should I do? Won't dev-kit fix anything that goes wrong?

Reply:
Ofc it will, but only when it gets executed again